spec/mailers/order_delivery_mailer_spec.rb

```diff
@@ -4,17 +4,44 @@
 
 RSpec.describe OrderDeliveryMailer, type: :mailer do
   describe 'Order delivery confirmation' do
-    let(:order) { create(:order) }
-    let(:mail) { described_class.with(order: order).delivery_email.deliver_now }
+    let!(:reward1) { create(:reward, delivery_method: 'post') }
+    let!(:order1) { create(:order, reward: reward1, reward_snapshot: reward1) }
+    let!(:address) { create(:address, order: order1) }
+    let(:mail1) { described_class.with(order: order1).delivery_by_post_email.deliver_now }
 
+    let!(:reward2) { create(:reward, delivery_method: 'online') }
+    let!(:order2) { create(:order, reward: reward2, reward_snapshot: reward2) }
+    let!(:reward_code) { create(:reward_code, reward: reward2, order: order2) }
+    let(:mail2) { described_class.with(order: order2).delivery_email.deliver_now }
+
+    # delivery by post
+    it 'checking the headers post email' do
+      order1.address = address
+      expect(mail1.subject).to have_content 'Your order has been delivered'
+      expect(mail1.to).to have_content order1.employee.email
+      expect(mail1.from).to have_content 'doboszmichal33@gmail.com'
```

Comment:
where this value is coming from?

Reply:
This value comes from: app/mailers/application_mailer.rb
My address is entered as the default value